### PR TITLE
feature/add consumer to check user

### PR DESCRIPTION
**Pull request by borovHokage**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/815

### CODE REVIEW

File: src/UserService.Broker/Consumers/CheckUserExistenceConsumer.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        {
+            var response = OperationResultWrapper.CreateResponse(GetUserExistenceInfo, context.Message);
+
+            await context.RespondAsync<IOperationResult<ICheckUserExistence>>(response);
+        }
+
+        private object GetUserExistenceInfo(ICheckUserExistence requestIds)
+        {
+            var userIds = _repository.AreExistingIds(requestIds.UserIds);
+
```
Comment (BelehovEgor) [inline src/UserService.Broker/Consumers/CheckUserExistenceConsumer.cs:28]: Don't use var here

File: src/UserService/appsettings.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
     "SearchUsersEndpoint": "UserService_SearchUsers",
     "CreateAdminEndpoint": "UserService_CreateAdmin",
     "FindParseEntitiesEndpoint": "UserService_FindParseEntities",
+    "CheckUserExistenceEndpoint": "UserService_CheckUserExistence",
```
Comment (BelehovEgor) [inline src/UserService/appsettings.json:42]: Maybe better users, we can check more than 1 user

File: src/UserService/Startup.cs [hunk 3]
```diff
@@ -145,9 +146,9 @@ private void ConfigureConsumers(IServiceCollectionBusConfigurator x)
                 ep.ConfigureConsumer<CreateAdminConsumer>(context);
             });
 
-            cfg.ReceiveEndpoint(rabbitMqConfig.FindParseEntitiesEndpoint, ep =>
```
Comment (BelehovEgor) [inline src/UserService/Startup.cs:148]: Why did you delete parse consumer?

**Review by BelehovEgor — CHANGES_REQUESTED** [review]

File: src/UserService.Broker/Consumers/CheckUserExistenceConsumer.cs (hunk 1 above)
Comment (borovHokage) [reply src/UserService.Broker/Consumers/CheckUserExistenceConsumer.cs:30]: done

File: src/UserService/Startup.cs (hunk 3 above)
Comment (borovHokage) [reply src/UserService/Startup.cs]: fixed

File: src/UserService/appsettings.json (hunk 2 above)
Comment (borovHokage) [reply src/UserService/appsettings.json]: done

**Review by MariyaSin — APPROVED** [review]

**Review by BelehovEgor — APPROVED** [review]

**Review by artshah-coder — APPROVED** [review]
OK

File: src/UserService.Data/UserRepository.cs [hunk 4]
```diff
@@ -129,6 +129,13 @@ public List<DbUser> Get(IEnumerable<Guid> userIds)
             return _provider.Users.Where(x => userIds.Contains(x.Id)).ToList();
         }
 
+        public List<Guid> AreExistingIds(List<Guid> userIds)
```
Comment (Glunky) [inline src/UserService.Data/UserRepository.cs:132]: mb rename it to getUserIds? firstly i thought that it should return bool

**Review by djino97 — APPROVED** [review]
